common/autoware_ad_api_specs/package.xml

```diff
@@ -13,6 +13,8 @@
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
   <buildtool_depend>autoware_cmake</buildtool_depend>
 
+  <depend>autoware_adapi_version_msgs</depend>
```

Comment:
No dependencies should be written on this package. This is because packages that use this package will have unnecessary dependencies. The packages that use this package write dependencies according to the headers that they use.

Reply:
Sorry. This comment is for https://github.com/autowarefoundation/autoware.universe/pull/3606#discussion_r1199956126.